app/params/app_settings.go

```diff
@@ -26,13 +26,8 @@ const (
 // Transactions with gas-price smaller than ProtocolMinGasPrice will fail during DeliverTx.
 var ProtocolMinGasPrice = sdk.NewDecCoinFromDec(BondDenom, sdk.MustNewDecFromStr("0.00"))
 
-func init() {
-	// XXX: If other upstream or external application's depend on any of Ojo's
-	// CLI or command functionality, then this would require us to move the
-	// SetAddressConfig call to somewhere external such as the root command
-	// constructor and anywhere else we contract the app.
+func SetAddressPrefixes() {
```

Comment:
Add a godoc here please, specifying the circumstance in which a package would want to add this to their init()